src/javascript/example/server/src/demo/demo.js

```diff
@@ -0,0 +1,96 @@
+import sessionless from 'sessionless-node';
+import chalk from 'chalk';
+import { associateKey, getUser, getUserByAssociatedKey, saveUser, getValue, saveValue } from '../persistence/user.js';
+
+export const addRoutes = (app) => {
+  app.post('/associate', async (req, res) => {
+    const payload = req.body;
+    const signature1 = payload.signature1;
+    const signature2 = payload.signature2;
+
+    const user = getUser(payload.uuid1);
+
+    if(!signature1 || !signature2) {
+      res.send(new Error('Need two signatures'));
+    }
+    
+    const message1 = JSON.stringify({
+      uuid: payload.uuid1,
+      timestamp: payload.timestamp1
+    });
+    
+    const message2 = JSON.stringify({
+      uuid: payload.uuid2,
+      timestamp: payload.timestamp2,
+      pubKey: payload.pubKey
+    });
+
+    if(!sessionless.verifySignature(signature1, message1, user.pubKey) || 
```

Comment:
There was a discussion of constructing messages to verify a signature. The message used to create the signature cannot differ from the one used to verify it. 
If we construct a message from the payload like in the line `17` and `22` the `verifySignature` function might fail depending of the JSON implementation. 
One end can encode the message as:
```json
{"uuid": "<val>", "timestamp": "<val>"}
```
and the second one as:
```json
{"timestamp": "<val>", "uuid": "<val>"}
```
The content is the same, but the message is different. I think we should highly avoid that.

The current body of this request looks like this:
```json
{
   "uuid1": "...",
   "timestamp1": "...",
   "signature1": "...",
   "uuid2": "...",
   "timestamp2": "...",
   "signature2": "...",
   "pubKey": "..."
}
```
We could encode each message into Base64 and then use the encoded string to create the signature. In practice that's how the body would look like now:
```json
{
   "message1": "eyJ1dWlkIjogInZhbDEiLCAidGltZXN0YW1wIjogInZhbDEifQ==",
   "signature1": "...",
   "message2": "eyJ1dWlkIjogInZhbDIiLCAidGltZXN0YW1wIjogInZhbDIifQ==",
   "signature2": "..."
   "pubKey": "..."
}
```
Where each `"message"` is encoded JSON: `{"uuid": "valX", "timestamp": "valX"}`.

Now it's possible to easily get the data and verify the signature.

The signing process could look like this:
```js
message = {
   uuid: "...",
   timestamp: "...",
};

messageEncoded = base64(message);

signature = sign(messageEncoded);

payload = {
   message: messageEncoded,
   signature: signature,
}
```